Source/WebCore/Modules/mediasource/SourceBuffer.cpp

```diff
@@ -697,12 +696,10 @@ void SourceBuffer::setActive(bool active)
         m_source->sourceBufferDidChangeActiveState(*this, active);
 }
 
-void SourceBuffer::sourceBufferPrivateDidReceiveInitializationSegment(InitializationSegment&& segment, CompletionHandler<void(ReceiveResult)>&& completionHandler)
+auto SourceBuffer::sourceBufferPrivateDidReceiveInitializationSegment(InitializationSegment&& segment) -> Ref<ReceiveResultPromise>
```

Comment:
Why auto for sourceBufferPrivateDidReceiveInitializationSegment and not for computeSeekTime (or the reverse)?

Reply:
Because it makes for a shorter writing :)

Otherwise I have to write the fully qualified type name